src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -62,6 +62,10 @@ def get(self, hostname: str) -> Optional[HostSpec]:
         hosts = [host for host in self.list() if host.hostname == hostname]
         return hosts[0] if hosts else None
 
+    @wait_api_result
+    def gather_facts(self, hostname: Optional[str] = None) -> List[Dict[str, Any]]:
+        return self.api.get_gather_facts(hostname)
```

Comment:
Yup, let's go with the `get_facts()` everywhere (I always find confusing when the same function is called `get_something()`, `something()`, ...).